docs/source/concepts/KafkaLiveStreams.rst

```diff
@@ -43,5 +43,58 @@ This following python script shows how this listener is used:
       Listener='KafkaHistoListener', Address='kafkabroker:address', AccumulationMethod='Replace',
       RunTransitionBehavior='Stop', OutputWorkspace='testout')
 
+Topic Configuration
+###################
+
+The topics that the listener will subscribe to are defined for a particular instrument in Mantid's "Facilities.xml" file. Here is the entry for the V20 beamline for example:
```

Comment:
Could you point to this in the source code maybe. The link is probably not fragile given that the location has remained unchanged in a long long time.